release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1135,6 +1140,13 @@ This release deprecates the `lastTriggeredImageID` in the `ImageChangeTrigger` o
 
 Therefore, update scripts and jobs that inspect `buildConfig.spec.triggers[i].imageChange.lastTriggeredImageID` accordingly. (link:https://issues.redhat.com/browse/BUILD-213[*BUILD-213*])
 
+[id="ocp-4-8-builds-jenkins-operator"]
+==== The Jenkins Operator is deprecated
+
+This release deprecates the Jenkins Operator. A future version of {product-title} will remove support for the Jenkins Operator and ignore it. Then, <describe of effects of removal>.
+
+Therefore, <describe alternatives and actions that users perform to use them>.
```

Comment:
@akram @gabemontero my suggestion:

```suggestion
Customers can continue to deploy Jenkins on {product-title} using the templates provided by the Samples Operator.
```

I don't think we should mention Helm at all until we have something concrete for customers to use.

Reply:
Updated.